apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts

```diff
@@ -26,99 +26,85 @@ export function Lti13CourseNavigationInstructor({
       </head>
       <body>
         ${Navbar({ resLocals, navPage: 'lti13_course_navigation' })} ${TerminologyModal()}
-        <script>
-          $(() => {
-            $('#connect_course_instance').one('change', () => {
-              $('#saveButton').prop('disabled', false);
-            });
-          });
-        </script>
-
         <main id="content" class="container mb-4">
           <h1>Welcome to PrairieLearn</h1>
           <p>
             To finish the integration for your course, we need to connect
```

Comment:
We don't use pronouns like "we" or "our" to refer to the platform anywhere else in the interface (as far as I know). I think we should continue with more impersonal tone:
```suggestion
            To finish the integration for your course, you need to connect
```